redfish-core/lib/power_subsystem.hpp

```diff
@@ -28,7 +29,7 @@ inline void
 inline void requestRoutesPowerSubsystem(App& app)
 {
     BMCWEB_ROUTE(app, "/redfish/v1/Chassis/<str>/PowerSubsystem/")
-        .privileges({{"Login"}})
+        .privileges(redfish::privileges::getPowerSubsystem)
```

Comment:
Why the change here?

Reply:
Done. I deleted it and add a Todo. @lkammath Please pick the latest patchset.